src/CoreWCF.NetTcp/tests/TransportFrameworkTests.cs

```diff
@@ -132,52 +142,64 @@ private void CheckIP(string ip)
         //
         // A succesful pass will include the client and origin matching and the
         // service and destination matching
-        private void CheckPort(string clientPort, string servicePort, Process myProcess)
+        private void CheckPort(ITestOutputHelper output, string clientPort, string servicePort, Process myProcess)
         {
             string line;
             string originPort;
             string destinationPort = null;
             string rawLine = null;
             string matchingPortRawLine = null;
             bool verifiedClientPort = false;
-
+            bool extraLogging = false;
+#if NETFRAMEWORK
```

Comment:
something like this would be more concise 
bool extraLogging = Environment.Version.Major == 4 && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)

Reply:
I know, this is quick and dirty. I plan to remove it at some point once I have confidence the issue is fixed.